linker-interface/shared/src/main/scala/org/scalajs/linker/interface/ModuleInitializer.scala

```diff
@@ -87,4 +89,38 @@ object ModuleInitializer {
         MethodName(mainMethodName, ArrayOfStringTypeRef :: Nil, VoidRef),
         args)
   }
+
+  private implicit object MethodNameFingerprint
+      extends Fingerprint[MethodName] {
+
+    override def fingerprint(methodName: MethodName): String =
+      methodName.simpleName.encoded.toString()
```

Comment:
Hum, this is plain wrong. A `MethodName` is not identified just by its simple name but by all of its members.

All of these should use `nameString` (which is defined and correct for both `ClassName` and `MethodName`).

Reply:
Fixed in dcb1a8ac1fa008d993c2fc107bbef72a1230688d